tests/scripts/thread-cert/v1_2_LowPower_7_2_01_ForwardTrackingSeries.py

```diff
@@ -0,0 +1,340 @@
+            .filter_mle_cmd(consts.MLE_LINK_METRICS_MANAGEMENT_REQUEST) \
+            .filter(lambda p: p.mle.tlv.link_sub_tlv == LinkMetricsSubTlvType.FORWARD_PROBING_REGISTRATION) \
+            .filter(lambda p: 0 in p.mle.tlv.link_forward_series) \
+            .filter(lambda p: p.mle.tlv.metric_type_id_flags.type is nullField) \
+            .filter(lambda p: p.mle.tlv.metric_type_id_flags.metric is nullField) \
+            .must_next()
+
+        # Step 10 - Leader responds with a Link Metrics Management Response
+        pkts.filter_wpan_src64(LEADER) \
+            .filter_wpan_dst64(SED_1) \
+            .filter_mle_cmd(consts.MLE_LINK_METRICS_MANAGEMENT_RESPONSE) \
+            .filter(lambda p: p.mle.tlv.link_status_sub_tlv == consts.LINK_METRICS_STATUS_SUCCESS) \
+            .must_next()
+
+        # Step 16 - SSED_1 requests a Forward Series Link Metrics by sending a Link Metrics Management Request
+        # Forward Series Flags = 0x02:
+        # - Bit 1 = 1, (all Link Layer Data Frames)
+        # - Bits 0, 2-7 = 0
+        # Concatenation of Link Metric Type ID Flags = 0x0a:
+        # - Item1: (0)(0)(001)(010) = 0x0a
+        # -- E = 0
+        # -- L = 0
+        # -- Type/Average Enum = 1 (Exponential Moving Average)
+        # -- Metrics Enum = 2 (Link Margin)
+        pkts.filter_wpan_src64(SSED_1) \
+            .filter_wpan_dst64(LEADER) \
+            .filter_mle_cmd(consts.MLE_LINK_METRICS_MANAGEMENT_REQUEST) \
+            .filter(lambda p: p.mle.tlv.link_sub_tlv == LinkMetricsSubTlvType.FORWARD_PROBING_REGISTRATION) \
+            .filter(lambda p: 2 in p.mle.tlv.link_forward_series) \
+            .filter(lambda p: consts.LINK_METRICS_TYPE_AVERAGE_ENUM_EXPONENTIAL in p.mle.tlv.metric_type_id_flags.type) \
+            .filter(lambda p: consts.LINK_METRICS_METRIC_TYPE_ENUM_LINK_MARGIN in p.mle.tlv.metric_type_id_flags.metric) \
+            .must_next()
+
+        # Step 17 - Leader responds with a Link Metrics Management Response
+        pkts.filter_wpan_src64(LEADER) \
+            .filter_wpan_dst64(SSED_1) \
+            .filter_mle_cmd(consts.MLE_LINK_METRICS_MANAGEMENT_RESPONSE) \
+            .filter(lambda p: p.mle.tlv.link_status_sub_tlv == consts.LINK_METRICS_STATUS_SUCCESS) \
+            .must_next()
+
+        # Step 19 - SSED_1 sends an MLE Data Request to retrieve aggregated Forward Series results
+        # - TLV Request TLV(Link Metrics Report TLV specified
+        # - Link Metrics Query TLV
+        # -- Link Metrics Query ID Sub-TLV
+        # --- Query ID: (ID set in step 16)
+        # - Verify that the Link Metrics Query Options Sub-TLV is NOT present
+        pkts.filter_wpan_src64(SSED_1) \
+            .filter_wpan_dst64(LEADER) \
+            .filter_mle_cmd(consts.MLE_DATA_REQUEST) \
+            .filter(lambda p: TlvType.LINK_METRICS_QUERY in p.mle.tlv.type) \
+            .filter(lambda p: p.mle.tlv.query_id == SERIES_ID_2) \
+            .must_next()
```

Comment:
`# - Verify that the Link Metrics Query Options Sub-TLV is NOT present` is not verified